packages/docs/content/foundation/typography/body.mdx

```diff
@@ -5,7 +5,7 @@ intro: Body text is used for the majority of the content on the website or appli
 core:
   githubLink: design-system/src/styles/base/_typography.scss
   sketchLink: WKOKl08
-  storybookLink: foundations-typography-bodytext--all-body-text
+  storybookLink: foundations-typography-body-text--all-body-text
```

Comment:
Navigating to Storybook from this page errored out for me. Should probably be `foundations-typography--body`

Reply:
This is what I see when I run storybook locally. Maybe I'm not understanding something?
![Screen Shot 2022-10-28 at 4 57 10 PM](https://user-images.githubusercontent.com/1449852/198731052-ad3168ed-0b85-43cb-a543-ec5138be3476.png)